.sync.yml

```diff
@@ -30,51 +30,9 @@ Gemfile:
       - gem: 'github_changelog_generator'
         version: '~> 1.15'
         condition: "Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.3.0')"
-      - gem: 'puppet-lint-file_ensure-check'
-        require: false
-      - gem: 'puppet-lint-manifest_whitespace-check'
-        require: false
-      - gem: 'puppet-lint-trailing_comma-check'
-        require: false
-      - gem: 'puppet-lint-optional_default-check'
-        require: false
-    ':test':
-      - gem: 'puppet_metadata'
```

Comment:
This is required and used by https://github.com/gibbs/puppet-auditd/blob/master/.github/workflows/ci.yml#L35

Reply:
I have re-added that gem